lib/is_bullshit.rb

```diff
@@ -13,10 +14,12 @@ class Object
     '0' => true,
     'nil' => true,
     'null' => true,
+    Float::INFINITY => true,
+    BigDecimal::INFINITY => true
   }
 
   def bullshit?
-    !!BULLSHIT_VALUES[self]
+    !!BULLSHIT_VALUES[self] or (self.nan? if [Float, BigDecimal].member? self.class)
```

Comment:
Won't this fail when looking for nil or null? I think you'll need to put that on a different line to allow `!!BULLSHIT_VALUES[self]` to continue to work